### PR TITLE
CIOEC GCcode presentation

### PR DESCRIPTION
**Pull request by smellems**
Related to #289

### CODE REVIEW

**Comment by remyb2canada** [comment]
Can you also use your Jekyll magic to change _includes/listPresentations.md?

It currently has hardcoded only the mandate.md one

**Comment by smellems** [comment]
Right, that is hardcoded.  Let's create an issue to see if we can list them using Jekyll.  I think I tried and it was easier to hardcode, but we should look again

**Comment by smellems** [comment]
@remyb2canada j'ai ajouté la boucle qui affichera les présentations avec Jekyll :smile:

**Comment by smellems** [comment]
Closes #345

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+---
+
+## Current State
+
+Each department manages its own internal VCS/Project management platforms
+
+> ESDC has TFS, SVN and GitLab (Azure)
+
+For open source code and working in the open, SaaS are recomended
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: recomended --> recommended

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+To increase sharing/reuse in the GC
+
+- Evolve GCcode as an officially supported internal VCS and project management platform for the GC community, with assistance from ESDC (e.g. funding)
+- Make it the default working environment for all source code projects in ESDC
+  - Internal to GC or private projets
+
+Not just a copy of the code, it's where teams work.
+
+A stepping stone to working in the open by default.
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: aka Inner Source : https://en.wikipedia.org/wiki/Inner_source

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- Wiki
+- Websites
+- Integration with other systems
+- ..
+
+See [Gitlab Docs](https://docs.gitlab.com/ee/README.html)
+
+---
+
+## Why not SaaS
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: The Pros and Cons seem to be inverted :thinking:

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+### Pros
+
+- **Leverage cloud infrastructure** - Don’t compete with providers' resources and ability to stay current
+- **Work in Complete Openness** - Allow employees to work in the open with the rest of the world
+- **Private repositories** - Private projects are available for free or as part of a subscription ($$)
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: Argh but we don't want private repos... Need to put some caveat (base cost is X but if you create private repos cost is `X + n * y` where `n` is the number of private repos and `y` the base cost per private repo...)

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+---
+
+## Benefits to ESDC – IITB
+
+- Strengthen **relationship** between IT and business through a common VCS and project management platform
+- Improve the **quality and security** of code with automated testing
+- **Accelerate** the rate of development and deployments
+- **Discover/reuse** code from other projects (incl. other depts)
+- Obtain metrics/statistics for **reporting and continuous improvement**
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: Work in the open, internally (related to relationship between IT and business where our work is fully visible, at least internally)

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+--
+
+### Benefits to ESDC – Existing Projects
+
+- Benefit from a **common** development platform:
+  - Functionalities covering the full development lifecycle (code, issues, tasks, Kanban, automated test and deployments)
+  - Adoption and training
+  - Centralized project communications
+- Enhance **planning** of features development through a single view
+- **Track time** spent on project tasks more efficiently
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: No more CATS!

File: presentations/en/GCcode-version-control-system-esdc-gc.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+- Ensure **accountability** through logging of changes and incidents
+- Improve code **quality** and **collaboration** by developing in the open, thereby reducing overall risk
+
+---
+
+## Proposal
+
+- Evolve GCcode as an officially supported internal VCS and project management platform for the GC community, with assistance from ESDC (e.g. funding)
+- Make it the default working environment for all source code projects in ESDC
```
Comment (gcharest) [inline presentations/en/GCcode-version-control-system-esdc-gc.md]: Both App Dev, Ops, Security, Compliance, Governance, etc.

File: presentations/en/GCcode-version-control-system-esdc-gc.md (hunk 1 above)
Comment (jaysonmc) [reply presentations/en/GCcode-version-control-system-esdc-gc.md]: If there are small changes that don't change the meaning in the initial PR, just update it :)

File: presentations/en/GCcode-version-control-system-esdc-gc.md (hunk 3 above)
Comment (gabrielcossette) [reply presentations/en/GCcode-version-control-system-esdc-gc.md]: Agreed, it wasn't clear. I changed to "Advantages of public SaaS" and vice-versa.

File: presentations/en/GCcode-version-control-system-esdc-gc.md (hunk 4 above)
Comment (gabrielcossette) [reply presentations/en/GCcode-version-control-system-esdc-gc.md]: But there's no base cost for the basic GitHub service?
Also, pricing is rather per user / month. So whenever a user needs to share a private repo with more than 3 collaborators, we need to pay.

**Pull request merged**